CRM/Case/Info.php

```diff
@@ -36,7 +36,7 @@ public function getInfo() {
       'translatedName' => ts('CiviCase'),
       'title' => ts('CiviCase Engine'),
       'search' => 1,
-      'showActivitiesInCore' => 0,
+      'showActivitiesInCore' => Civi::settings()->get('civicaseShowCaseActivities') ?? 0,
```

Comment:
only with `'showActivitiesInCore' => Civi::settings()->get('civicaseShowCaseActivities')` should be fine.

Reply:
When its installed it should pick the default from [here](https://github.com/civicrm/civicrm-core/pull/16360/files#diff-6419f840391ad2d06092d9dd08faff03R108).